app/components/gh-member-settings-form.js

```diff
@@ -45,8 +45,14 @@ export default class extends Component {
     }
 
     get products() {
-        let products = this.member.get('products') || [];
         let subscriptions = this.member.get('subscriptions') || [];
+        // Create the products from `subscriptions.price.product`
+        let products = subscriptions
```

Comment:
This change causes a regression [here](https://github.com/TryGhost/Admin/blob/0ab6acd8d9dd43686e843ccb7c166f7c0d25592c/app/components/gh-member-settings-form.hbs#L69), because products is no longer empty - which means there's no option to give the member a comped subscription.

Reply:
@tpatel we can reinstate the "Add complimentary plan" (not exact wording) functionality using a similar pattern to the one used in the UI for Tiers:

![image](https://user-images.githubusercontent.com/3798302/157715192-1e03f965-09c9-441d-bdb0-c482a6ce4394.png)